### PR TITLE
Adds asthmosthia.dm. Symptom that creates atmospheric gases

### PR DESCRIPTION
**Pull request by LanCartwright**
Changes it a little from the old version

### CODE REVIEW

**Comment by GunHog** [comment]
Well, I guess it has a use as a de-stealthing agent for the mixing process...

**Comment by Buggy123** [comment]
:+1:

**Comment by LanCartwright** [comment]
This was tested.
It had to be buffed due to the different atmospheric systems that contrasted from the original virus.

There is an additional virus unobtainable outside of normal means, which is meant for traitors only. This will be added later. This virus is really plain if purchased and thrown to infect people (salt will cure it) and is meant for players to build upon it.

**Comment by PKPenguin321** [comment]
RIP

File: code/datums/diseases/advance/symptoms/asthmothia.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	resistance = -5
+	stage_speed = -5
+	transmittable = -6
+	level = 5
+	severity = 5
+
+/datum/symptom/asthmothia/Activate(var/datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/turf/simulated/T = get_turf(A.affected_mob)
```
Comment (AnturK) [inline code/datums/diseases/advance/symptoms/asthmothia.dm]: What about space.

**Comment by AnturK** [comment]
You got some rogue config changes in this branch.

**Comment by LanCartwright** [comment]
yeah I don't know what the hell is gone on. every time it happens.

**Comment by duncathan** [comment]
@LanCartwright

**Comment by KorPhaeron** [comment]
The PR is fucked and viro just got major changes anyway, remake this later when we figure out where viro is in a couple weeks

**Pull request closed without merging**